streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -581,11 +581,32 @@ public KafkaStreams(final Topology topology,
         this(topology.internalTopologyBuilder, config, clientSupplier);
     }
 
+    /**
+     * Create a {@code KafkaStreams} instance.
+     *
+     * @param topology       the topology specifying the computational logic
+     * @param config         the Kafka Streams configuration
+     * @param time           {@code Time} implementation; cannot be null
+     * @throws StreamsException if any fatal error occurs
+     */
+    public KafkaStreams(final Topology topology,
```

Comment:
Yes, see my description on the PR.

Reply:
You mentioned in the PR description that `Time` was added for the convenience of unit testing, should we add something similar to javadoc?

Reply:
We usually do not need to mention that, since people may have their own `Time` implementation other than `SYSTEM` in their applications that is not only for testing.